iogt/settings/base.py

```diff
@@ -38,6 +38,7 @@
 INSTALLED_APPS = [
     'home',
     'search',
+    'iogt',
```

Comment:
This is unusual and I don't think we should be doing it. 'iogt' is a project, not an app.